html-css/week2/Styles.css

```diff
@@ -0,0 +1,269 @@
+* {
+    box-sizing: border-box;
+    font-family: "Roboto", sans-serif;
+}
+
+body{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    background-color: #ffffffb1;
+    width: 100%;
+    height: 100%;
+}
+
+header{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    box-shadow: 0px -1px 0px 0px #241C154D inset;
+    width: 1440px;
+    height: 177px;
+    background: #F6F6F4;
+    position:relative;
+}
+
+.searchbar {
+    display: flex;
+    flex-direction: column;
+    justify-content:flex-end;
+    align-items: center;
+    max-width: 680px;
+}
+
+.search{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    height: 48px;
+    width: 680px;
+    padding: 1px;
+    background-color: #fff;
+    border-radius: 8px;
+    position: relative;
+    border: 2px solid #007c89;
+    overflow: hidden;
+}
+
+.fa-magnifying-glass {
+    position: left;
+    color: #484848;
+    left: 10px;
+    font-size: 16px;
+    cursor: pointer;
+
+}
+.fa-xmark {
+    position: absolute;
+    right: 10px;
+    font-size: 18px;
+    width: 15.49px;
+    height: 15.49px;
+    color: #484848;
+    cursor: pointer;
+}
+
+input {
+    flex: 1;
+    border: 1px solid white;
+    padding: 10px;
+    font-size: 16px;
+}
+
+.navbar{
+    display: flex;
+    flex-direction: row;
+    align-items: flex-end;
+    padding: 6px;
+    flex-wrap: nowrap;
+    
+    li{
+        list-style: none;
+    }
+
+    a {
+        text-decoration: none;
+        color: #6d6863;
+        font-size: 16px;
+
+    }
+
+    a:hover {
+        color: #007c89;
+        font-weight: 500;
+        border-bottom: 3px solid #007c89;
+        padding-bottom: 16px;
+    }
+
+    .active {
+        color: #007c89;
+        font-weight: 500;
+        border-bottom: 3px solid #007c89;
+        padding-bottom: 16px;
+    }
+}
+.container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    max-width: 680px;
+    width: 100%;
+}
+.find-results{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    padding: 24px 10px 24px 0;
+    border-bottom: 1px solid #e7e6e5;
+    gap: 70px;
+    color: #241c15;
+}
+/* i can't make this section*/
+.language{
+    display: flex;
+    flex-direction: row;
+    font-size: 16px;
+    font-weight: 500;
+    gap: 10px;
+}
+.results {

```

Comment:
The styles of the results are not quite similar to the design. In a job, you will be asked to make it look as close as you can to a design.